agent/rpc_from.go

```diff
@@ -18,14 +18,14 @@ func (a *orbAgent) handleGroupMembership(rpc fleet.GroupMembershipRPCPayload) {
 	if rpc.FullList {
 		a.unsubscribeGroupChannels()
 		a.subscribeGroupChannels(rpc.Groups)
+		err := a.sendAgentPoliciesReq()
+		if err != nil {
+			a.logger.Error("failed to send agent policies request", zap.Error(err))
+		}
 	} else {
 		// otherwise, just add these subscriptions to the existing list
 		a.subscribeGroupChannels(rpc.Groups)
 	}
-	err := a.sendAgentPoliciesReq()
```

Comment:
Doesn't moving this mean we will not ask for policies unless we get the FullList of groups, and if so is that right?